src/app/event-card/event-card.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from "@angular/core";
+import { EVENTS } from "../example-events";
+
+@Component({
+  selector: "app-event-card",
+  templateUrl: "./event-card.component.html",
+  styleUrls: ["./event-card.component.scss"]
+})
+export class EventCardComponent implements OnInit {
+
+  @Input() index!: number;
+  events = EVENTS;
+  constructor() { }
```

Comment:
you should declare the index in the constructor to set a default property value to it aka `index = 0;`